docs/api/commands/getallsessionstorage.mdx

```diff
@@ -56,15 +56,15 @@ set to `value2` on `https://other.com`, `cy.getAllSessionStorage()` will yield:
 ### Get all sessionStorage
 
 ```javascript
-cy.visit('https://example.com', {
+cy.visit('/users', {
   onBeforeLoad(win) {
     win.sessionStorage.setItem('key', 'value')
   },
 })
 
 cy.getAllSessionStorage().then((result) => {
   expect(result).to.deep.equal({
-    'https://example.com': {
+    '/users': {
```

Comment:
```suggestion
    'http://localhost:8080': {
```

Same as with the local storage version.

Reply:
It won't let me leave another code suggestion, but this should be: 

```js
cy.visit('https://example.cypress.io', {
  onBeforeLoad(win) {
    win.sessionStorage.setItem('key', 'value')
  },
})

cy.getAllSessionStorage().then((result) => {
  expect(result).to.deep.equal({
    'https://example.cypress.io': {
      key: 'value',
    },
    },
  })
})
```

The keys for the result of `cy.getAllSessionStorage()` are the full origins, not the paths.